### DISCUSSION TITLE
Welcome to amdgpud Discussions!

### DISCUSSION THREAD

**Discussion by Eraden** [Eraden/amdgpud#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VitaliyKulikov** [comment]
hello, my name is Vitaliy and I am awesome pal ))
so, i have some issue with new version #27 and following idea:
- as a `noise hater` i would like to optimise `fan curve` so that no noise around ))
-- possibility to dump metrics to csv file, like: current time, current temperature, current speed, current matrix speed limiter, current matrix temperature limiter
-- dump can be done every second or with internal cycle 
-- same temperature value (int) with previous can be omitted

thanks

**Reply by Eraden** [reply]
I'll add such metrics in version 1.0.8

**Reply by Eraden** [reply]
Current release was just hotfix for bug and small roadmap

**Reply by VitaliyKulikov** [reply]
Thanks!

**Reply by Eraden** [reply]
I'm thinking about extracting monitor feature to binary Daemon which will have an option to write to file.
